doc/mesh_filter/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY meshes DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY urdf DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY rviz DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
Currently, the install folder of moveit_tutorials looks like this:
```
❯ ls install/share/moveit_tutorials/
cmake  doc  launch  package.xml
```
where the `doc` folder already contains subfolders for some tutorials with auxiliary data.
I would propose to leave `meshes`, `urdf` and `rviz` files there as well, only launch files are grouped together in one folder.
If you want, I can do these changes myself.

Reply:
Incorporated the changes you recommended